Sources/UIKit/UIViewController.swift

```diff
@@ -32,8 +32,8 @@ public extension UIViewController {
     ///   - unselectedRenderMode: UIImageRenderingMode to the unselected image. Default .alwaysOriginal
```

Comment:
Documentation is still referencing old types.

Reply:
Nice catch! 👍